rdmo/projects/tests/test_viewset_project_page.py

```diff
@@ -45,3 +48,24 @@ def test_detail(db, client, username, password, project_id, page_id):
         assert response.json().get('id') == page_id
     else:
         assert response.status_code == 404
+
+
+@pytest.mark.parametrize('username,password', users)
```

Comment:
not for every user and project, please.

Reply:
Ive used now only the first with `[0:1]`, is ok ?

Reply:
why not remove the parametrization for users and projects and use `user` and `1` explicitly?

Reply:
is also possible.. hard coded tests look so strange, they are all parametrized..

Reply:
now its there just: `test_detail_order_in_page(db, client)`